### ISSUE TITLE
In Solus Budgie linux in Alt+Tab menu the KeeWeb icon is misisng.

### ISSUE THREAD

**Issue by paul-alkhimov** [keeweb/keeweb#966]
In Solus Budgie when switching the apps with Alt+Tab, the KeeWeb icon is missing.

[screenshot](https://www.dropbox.com/s/ulmia91i4mejnwq/Screenshot%20from%202018-08-01%2010-39-09.png?dl=0)

KeeWeb v1.6.3 (cded8a4, 2017-12-11)
Environment: electron v1.7.9
User-Agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) KeeWeb/1.6.3 Chrome/58.0.3029.110 Electron/1.7.9 Safari/537.36

App log:

[DEBUG] 2018-08-01T09:08:11.641Z [plugin-gallery] Loaded cached plugin gallery 34ms
[INFO ] 2018-08-01T09:08:11.646Z [plugin:keewebhttp] Loading plugin with resources js (local)
[DEBUG] 2018-08-01T09:08:11.648Z [storage-plugin-files] Load keewebhttp_plugin.js
[DEBUG] 2018-08-01T09:08:11.651Z [storage-plugin-files] Loaded keewebhttp_plugin.js 3ms
[DEBUG] 2018-08-01T09:08:11.651Z [plugin:keewebhttp] Resource data loaded js 5ms
[DEBUG] 2018-08-01T09:08:11.653Z [plugin:keewebhttp] Resource signature validated js 1ms
[INFO ] 2018-08-01T09:08:11.653Z [plugin:keewebhttp] Installing plugin resources
[DEBUG] 2018-08-01T09:08:11.684Z [plugin:keewebhttp] Plugin script installed 4ms
[DEBUG] 2018-08-01T09:08:11.685Z [plugin:keewebhttp] Plugin settings loaded 0ms
[INFO ] 2018-08-01T09:08:11.685Z [plugin:keewebhttp] Install complete 39ms
[INFO ] 2018-08-01T09:08:11.687Z [plugin-mgr] Loaded 1 plugins 81ms
[INFO ] 2018-08-01T09:08:12.303Z [app] Started in 1819ms ¯\_(ツ)_/¯
[INFO ] 2018-08-01T09:08:12.305Z [updater] Next update check will happen in 86075s
[DEBUG] 2018-08-01T09:08:12.391Z [keewebhttp] Server running at http://127.0.0.1:19455/
[INFO ] 2018-08-01T09:08:26.051Z [open:########] File open request
[INFO ] 2018-08-01T09:08:26.051Z [open:########] Open file from cache, will sync after load dropbox
[DEBUG] 2018-08-01T09:08:26.051Z [storage-cache] Load #####
[DEBUG] 2018-08-01T09:08:26.055Z [storage-cache] Loaded ##### 2ms
[INFO ] 2018-08-01T09:08:26.055Z [open:########] Loaded file from cache null
[INFO ] 2018-08-01T09:08:27.374Z [file] Opened file ########: 1317ms, 65536 rounds, 424 kB
[DEBUG] 2018-08-01T09:08:27.375Z [app] Add last open file ##### ######## dropbox /########.kdbx ########
[INFO ] 2018-08-01T09:08:27.562Z [open:########] Sync just opened file
[INFO ] 2018-08-01T09:08:27.584Z [sync:########] Sync started dropbox /########.kdbx {}
[INFO ] 2018-08-01T09:08:27.586Z [sync:########] Stat file
[DEBUG] 2018-08-01T09:08:27.587Z [storage-dropbox] Stat /########.kdbx
[DEBUG] 2018-08-01T09:08:27.978Z [storage-dropbox] Stated /########.kdbx ######## 391ms
[INFO ] 2018-08-01T09:08:27.978Z [sync:########] Stat found same version, not modified
[INFO ] 2018-08-01T09:08:27.978Z [sync:########] Sync finished no error

**Comment by antelle** [comment]
This seems to be an electron issue, not sure if we can fix it

**Comment by paul-alkhimov** [comment]
Then close it to clean the tracker.

**Comment by antelle** [comment]
I'm not sure if it's electron, because I don't see it in their issues

**Comment by paul-alkhimov** [comment]
Yes, exactly this one is not there, but they do have some [problems](https://github.com/electron/electron/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+linux+icon) with app icon

**Comment by antelle** [comment]
Maybe updating to new electron will also help

**Comment by paul-alkhimov** [comment]
yeah, meanwhile I check https://budgie-desktop.org/home/

**Comment by antelle** [comment]
Interesting, thanks, then it's possible to fix
